src/main/scala/com/raphtory/core/components/graphbuilder/GraphBuilder.scala

```diff
@@ -9,6 +9,69 @@ import com.typesafe.config.Config
 import scala.collection.mutable
 import scala.collection.mutable.ArrayBuffer
 
+/**
+  * {s}`GraphBuilder`
+  *
+  * : `GraphBuilder` trait for creating a Graph by adding and deleting vertices and edges.
+  *    While using `GraphBuilder`, override {s}`parseTuple(tuple: T)` to define parsing of rows and use methods {s}`addVertex`, {s}`addEdge`
+  *    for adding vertices and edge to the graph. Deletion can be performed with {s}`deleteEdge` and {s}`deleteVertex` methods.
+  *
+  * ## Methods
+  *
+  *    {s}`addVertex(updateTime: Long, srcId: Long)`
+  *      : Adds vertex for a timestamp {s}`updateTime` and ID {s}`srcId`
```

Comment:
yes this adds a vertex, but it does more, for example it does not incvlude properties and it does not include the type? but what does add vertex actually do? it adds  the object `addVertex` to the updates.